### PR TITLE
[v7.4.x] Expressions: Add option to disable feature

### PR DESCRIPTION
**Pull request by grafanabot**
Backport 9ada4b605205744329e0be26a4a0512d02436da4 from #30541

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -893,3 +893,7 @@ use_browser_locale = false
 
 # Default timezone for user preferences. Options are 'browser' for the browser local timezone or a timezone name from IANA Time Zone database, e.g. 'UTC' or 'Europe/Amsterdam' etc.
 default_timezone = browser
+
+[expressions]
+# Disable expressions & UI features
```
Comment (oddlittlebird) [inline conf/defaults.ini:898]:
```suggestion
# Turn the expressions functionality on or off.
```

File: conf/sample.ini [hunk 2]
```diff
@@ -883,3 +883,7 @@
 
 # Default timezone for user preferences. Options are 'browser' for the browser local timezone or a timezone name from IANA Time Zone database, e.g. 'UTC' or 'Europe/Amsterdam' etc.
 ;default_timezone = browser
+
+[expressions]
+# Disable expressions & UI features
```
Comment (oddlittlebird) [inline conf/sample.ini:888]:
```suggestion
# Turn the expressions functionality on or off.
```

File: docs/sources/administration/configuration.md [hunk 3]
```diff
@@ -1505,3 +1505,8 @@ Set this to `true` to have date formats automatically derived from your browser
 ### default_timezone
 
 Used as the default time zone for user preferences. Can be either `browser` for the browser local time zone or a time zone name from the IANA Time Zone database, such as `UTC` or `Europe/Amsterdam`.
+
+## [expressions]
+>Note: This is available in Grafana v7.4 and later versions.
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:1510]:
```suggestion
> **Note:** This feature is available in Grafana v7.4 and later versions.
```

File: docs/sources/administration/configuration.md [hunk 4]
```diff
@@ -1505,3 +1505,8 @@ Set this to `true` to have date formats automatically derived from your browser
...
+### enabled
+Set this to `false` to disable expressions and hide them in the Grafana UI. Default is `true`.
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:1512]:
```suggestion

Set this to `false` to turn off expressions and hide them in the Grafana UI. Default is `true`.
```

**Review by oddlittlebird — COMMENTED** [review]
The description in the .ini files did not make sense to me, so I suggested new verbiage.

**Comment by papagian** [comment]
> The description in the .ini files did not make sense to me, so I suggested new verbiage.

This PR is only for back porting the modifications in v7.4.x branch. The original PR #30541 was approved and already merged in the master. Therefore for fixing the docs we have to merge this and create a new one with the suggested modifications (to be merged in master and then in v7.4.x).

**Review by oddlittlebird — APPROVED** [review]
Understood. Are you willing to file the PR?

**Comment by achatterjee-grafana** [comment]
@oddlittlebird, I approved PR #30541. What changes are needed for that? Can you please clarify. Thanks!

**Comment by oddlittlebird** [comment]
Just some minor editing suggestions, see comments above. This PR is good to go.

**Comment by papagian** [comment]
> Understood. Are you willing to file the PR?

#30589

**Review by kylebrandt — APPROVED** [review]

**Pull request merged**